epos:
  ros__parameters:
    device_name: "EPOS4"
    protocol: "MAXON SERIAL V2"
    interface: "USB"
    portname: "USB0"

    use_sub_device: false

    auto_enable: true
    dataout_freq: 10

    motor_id: [0]
